app/controllers/donations_controller.rb

```diff
@@ -0,0 +1,42 @@
+class DonationsController < ApplicationController
+  def new
+    @html_id = 'page'
+    @body_id = 'support'
+    @title   = I18n.t('views.donations.support.heading')
+  end
+
+  def thanks
+    @html_id = 'page'
+    @body_id = 'support'
+    @title   = I18n.t('views.donations.thanks.heading')
+  end
+
+  def create
+    amount = params[:amount].tr('$', '')
+
+    customer = Stripe::Customer.create(
```

Comment:
I am having a hard time figuring out the answer to this from the stripe docs, so maybe i will just experiment, but:

Is this a `find_or_create` type operation? if someone donates a 1-time donation, we add them as a customer. If they come back to do another donation, will we get an error, or will everything work fine?

Reply:
This came up while @veganstraightedge and I were working through some of this and we decided to just create new customers each time. If someone wants to give a one time donation and then create a monthly one, we'll just create a new customer. If that gets out of hand and we get a bunch of duplicate customers then maybe we can adjust that in the future. Personally, not too worried about it.